js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+        const request = {
+            'txid': id,
+        };
+        const response = await this.privateGetDeposit (this.extend (request, params));
+        // {
+        //   "id": 4831985,
+        //   "currency": "ltc",
+        //   "amount": "0.14108256",
+        //   "fee": "0.0",
+        //   "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //   "uid": null,
+        //   "created_at": "2022-02-01T18:10:32+03:00",
+        //   "confirmations": "6",
+        //   "done_at": null,
+        //   "state": "accepted"
+        // }
+        // const data = this.safeValue (response, 'data', {});
+        // const deposit = this.safeValue (data, 'deposit', {});
+        return this.parseTransaction (response);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+        }
+        const response = await this.privateGetDeposits (params);
+        // const data = this.safeValue (response, 'data', {});
+        // const deposits = this.safeValue (response);
+        // [
+        //     {
+        //         "id": 4831985,
+        //         "currency": "ltc",
+        //         "amount": "0.14108256",
+        //         "fee": "0.0",
+        //         "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //         "uid": null,
+        //         "created_at": "2022-02-01T18:10:32+03:00",
+        //         "confirmations": "6",
+        //         "done_at": null,
+        //         "state": "accepted"
+        //     }
+        // ]
+        return this.parseTransactions (response, currency, since, limit);
+    }
+
+    parseOrder (order, market = undefined) {
+        // [
+        //   {
+        //     "id": 287792646,
+        //     "at": 1643956089,
+        //     "side": "sell",
+        //     "ord_type": "limit",
+        //     "price": "0.141",
+        //     "avg_price": "0.0",
+        //     "state": "cancel",
+        //     "market": "ltcbtc",
+        //     "created_at": "2022-02-04T09:28:09+03:00",
+        //     "volume": "0.141",
+        //     "remaining_volume": "0.141",
+        //     "executed_volume": "0.0",
+        //     "trades_count": 0,
+        //     "strategy": null
+        //   }
+        // ]
+        const id = this.safeString (order, 'id');
+        const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
+        const sideType = this.safeString (order, 'ord_type');
+        const side = this.safeString (order, 'side');
+        const price = this.safeString (order, 'price');
+        const amount = this.safeString (order, 'avg_price');
+        const remaining = this.safeString (order, 'remaining_volume');
+        // const open = this.safeValue (order, 'open', false);
```

Comment:
remove these comments pls

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/commit/bb2b0b2ef29c557eee9dad97fee9c010de932c9a